iso.bbx

```diff
@@ -606,6 +598,41 @@
 
 % BIBLATEX CORE ADJUSTMENTS
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Localisation strings in the main document language
+%
+% \mainlangbibstring in place of \bibstring
+% \mainlstring in place of \biblstring
+% \mainsstring in place of \bibsstring
+
+% [<wrapper>]{<string>}
+\newrobustcmd*{\blx@imc@mainlangbibstring}[2][\@firstofone]{%
+  \blx@mainlangbibstring{#1}{\abx@str}{#2}}
+
+\newrobustcmd*{\blx@imc@mainlstring}[2][\@firstofone]{%
+  \blx@mainlangbibstring{#1}{abx@lstr}{#2}}
+
+\newrobustcmd*{\blx@imc@mainsstring}[2][\@firstofone]{%
+  \blx@mainlangbibstring{#1}{abx@sstr}{#2}}
+
+\protected\def\blx@mainlangbibstring#1#2#3{%
+  {\mainlang% <---- use main document language
+  \blx@begunit
+  \blx@hyphenreset
+  \blx@bibstringsimple
+  \lowercase{\edef\blx@tempa{#3}}%
+  \ifcsundef{#2@\blx@tempa}
+    {\blx@warn@nostring\blx@tempa
+     \blx@endnounit}
+    {\blx@imc@ifcapital
+       {#1{\MakeCapital{\csuse{#2@\blx@tempa}}}}
+       {#1{\csuse{#2@\blx@tempa}}}%
+     \blx@endunit}}}
+
+\blx@regimcs{%
+  \mainlangbibstring \mainlstring \mainsstring
```

Comment:
Are the names `\mainlangbibstring`, `\mainlstring`, `\mainsstring` fixed for backwards compatibility reasons? If not I think it would be nicer to unify them to
```latex
  \mainlangbibstring \mainlangbiblstring \mainlangbibsstring
```